runner/file.go

```diff
@@ -119,7 +136,11 @@ func transformCSV(in *bufio.Reader, out *bufio.Writer, delimiter rune) error {
 				continue
 			}
 
-			recordToMap(row, &fields, record)
+			if convertNumbers {
+				recordToMap(row, &fields, convertNumbersInRecord(record))
+			} else {
+				recordToMap(row, &fields, record)
+			}
```

Comment:
I think we can do the convertNumbersInRecord function *inside* of recordToMap which also iterates over all fields.